web/modules/custom/rdf_schema_field_validation/rdf_schema_field_validation.install

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update hooks of the rdf_schema_field_validation module.
+ */
+
+use Drupal\rdf_entity\Entity\RdfEntityMapping;
+use EasyRdf\Graph;
+use EasyRdf\GraphStore;
+
+/**
+ * Implements hook_update_N().
+ */
+function rdf_schema_field_validation_update_8102() {
+  $graph_uri = 'http://adms-definition';
+  $class_definition = 'http://www.w3.org/2000/01/rdf-schema#Class';
+
+  $sparql_endpoint = \Drupal::service('sparql_endpoint');
+  $connection_options = $sparql_endpoint->getConnectionOptions();
+  $connect_string = 'http://' . $connection_options['host'] . ':' . $connection_options['port'] . '/sparql-graph-crud';
+  // Use a local SPARQL 1.1 Graph Store.
+  $gs = new GraphStore($connect_string);
+  $graph = new Graph();
+  try {
+    $graph->parseFile(DRUPAL_ROOT . '/../resources/fixtures/adms-definition.rdf');
+  }
+  catch (\Exception $e) {
+    throw $e;
```

Comment:
This is weird :) In fact all this `try {...} catch() {...}` block can be rewritten as 
```php
$graph->parseFile(DRUPAL_ROOT . '/../resources/fixtures/adms-definition.rdf');
```
I know, the IDE. Ignore it

Reply:
No the IDE was not complaining on that. I don't remember why I had the try-catch. I am going to remove it because anyway, I am not cancelling any operation so it does not make sense.